tests/test_custom_models.py

```diff
@@ -127,10 +127,20 @@
         "Vanilla MLP 1 LoRA with lora_b bias",
         "MLP",
         LoraConfig,
-        {"target_modules": ["lin0", "lin1"], "lora_bias": True},
+        {"target_modules": ["lin0", "lin1"], "lora_bias": True, "lora_alpha": 32},
```

Comment:
Perhaps I would add a comment to reflect the note you mentioned in the OP.

Reply:
Good point, I added a comment:

> # It's important to set lora_alpha != r to ensure that scaling is taken into account correctly